src/commands/create.ts

```diff
@@ -32,51 +32,34 @@ export const builder: CommandBuilder<Options, Options> = (yargs) =>
     }) as yargs.Argv<Options>;
 
 export const handler = async (argv: Arguments<Options>): Promise<void> => {
-  const { schema, prefix, privateKey, chain, alchemy, infura, etherscan } =
-    argv;
-
-  if (!privateKey) {
-    console.error("missing required flag (`-k` or `--privateKey`)\n");
-    process.exit(1);
-  }
-  const network = getChains()[chain];
-  if (!network) {
-    console.error("unsupported chain (see `chains` command for details)\n");
-    process.exit(1);
-  }
+  const {
+    schema,
+    prefix,
+    privateKey,
+    chain,
+    alchemy,
+    infura,
+    etherscan,
+    rpcRelay,
+  } = argv;
 
-  const wallet = new Wallet(privateKey);
```

Comment:
This has all been extracted into a single getWallet function to DRY things out a bit.